src/nedap/speced/def/impl/spec_assertion/cljs.cljc

```diff
@@ -0,0 +1,10 @@
+(ns nedap.speced.def.impl.spec-assertion.cljs
+  (:require
+   [cljs.test :as test]
+   #?(:clj [nedap.speced.def.impl.spec-assertion :as impl.spec-assertion])))
+
+#?(:clj
+   (defmethod test/assert-expr 'spec-assertion-thrown? [_env msg form]
+     (clojure.core/let [spec-sym (second form)
```

Comment:
y the explicit ns in `clojure.core/let` ?

Reply:
Most likely accidental. It's better expressed as `cljs.core/let` anyway here ^^

(both work - cljs can be lenient w/ these)